config.template.yml

```diff
@@ -151,6 +151,9 @@ authentication_backend:
     # one returned by the LDAP server is used.
     mail_attribute: mail
 
+    # The attribute holding the display name of the user. This will be used to greet an authenticated user.
+    name_attribute: displayname
```

Comment:
What do you think about using `display_name_attribute` as the config key to make it more obvious that it's not a first name nor a last name but the actual name that will be displayed. Personally, I find it more explicit and if we add other name attributes in the future (first, last for whatever reason), the distinction will already be made without breaking change.

Reply:
I'm not particularly fussed either way if we want it to be more explicit happy to rename it to `display_name_attribute` though I think that's a little long and wordy, but I can definitely see your thought process here.

If we decide to introduce split attributes first and/or last names I feel like they would be better suited to be prefixed.

Reply:
I think the fact that it's wordy is not a big deal as the admin will only read it once in a while, it's not like it's modified so often. But the advantage of it being explicit is that it should be clear in the admin's mind what this attribute is used for by simply reading it. It's also clearer to devs what this variable is supposed to be used for in the code.